src/mixins/watchers.js

```diff
@@ -12,6 +12,9 @@ export default {
   },
   methods: {
     queueWatcherSync(watcher) {
+      //add a random sufix to uniquely identify this watcher call
```

Comment:
Could you try https://www.npmjs.com/package/uuid for a more robust solution?

Reply:
The unique ui is per watcher and time (milliseconds) so it seems a good solution in this context, in that way we avoid to use an extra module for just this.